feature-libs/customer-ticketing/components/customer-ticketing-details/customer-ticketing-details.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit } from '@angular/core';
+import { TranslationService } from '@spartacus/core';
+import {
+  COLOR_CLOSE,
+  COLOR_OPEN,
+  STATUS_CLOSE,
+  STATUS_OPEN,
+} from '@spartacus/customer-ticketing/root';
+import { Card } from '@spartacus/storefront';
+import { Observable, of } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-customer-ticketing-details',
+  templateUrl: './customer-ticketing-details.component.html',
+})
+export class CustomerTicketingDetailsComponent implements OnInit {
+  constructor(protected translation: TranslationService) {}
+
+  ticketDetails$ = of({
```

Comment:
👍 

no need to mock data now that we have a way forward on what to do to get the data from the web server

Reply:
maybe it's best to work on the core part first than the UI now. What do you think? 

If no, then continue on as I don't want to say how to work haha. Just a suggestion

Reply:
yes yes, I also prefer working on core first, but the UI is 90% done including the messaging and reopen/close request.
So, we thought we can merge this changes into epic and start on the core now.